controllers/mirrorpeer_controller.go

```diff
@@ -878,7 +878,7 @@ func (r *MirrorPeerReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Watches(&corev1.ConfigMap{}, handler.EnqueueRequestsFromMapFunc(cmToMirrorPeerMapFunc),
 			builder.WithPredicates(predicate.NewPredicateFuncs(func(object client.Object) bool {
 				cm, ok := object.(*corev1.ConfigMap)
-				if !ok || cm.Name != utils.ClientInfoConfigMapName || cm.Namespace != r.CurrentNamespace {
+				if !ok || (cm.Name != utils.ClientInfoConfigMapName && cm.Name != utils.RamenHubOperatorConfigName) || cm.Namespace != r.CurrentNamespace {
```

Comment:
I don't think we need to check if it's a configmap or not.
Can we have it structured like: https://github.com/red-hat-storage/ocs-operator/blob/main/controllers/storagecluster/storagecluster_controller.go#L235-L249

```suggestion
builder.WithPredicates(
    predicate.Or(
      util.NamePredicate(utils.ClientInfoConfigMapName),
      util.NamePredicate(utils.RamenHubOperatorConfigName),
   ),
   util.NamespacePredicate(r.CurrentNamespace),
   util.EventTypePredicate(true,true,true,true),
)
```

Eample of predicate utils: https://github.com/red-hat-storage/ocs-operator/blob/main/controllers/util/predicates.go#L74-L107

Reply:
Sounds good, updated the code to use predicates to filter it.
PTAL.